portugues/titanic/pular/boiar/computador.md

```diff
@@ -0,0 +1,7 @@
+As pessoas não entedem muito bem como um computador chegou a costa em cima de um pedaco de madeira ainda funcionando.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.